webapp/pages/settings/index.vue

```diff
@@ -88,8 +88,8 @@ export default {
     return {
       axiosSource: null,
       cities: [],
-      sending: false,
-      loading: false,
+      loadingData: false,
+      loadingGeo: false,
```

Comment:
This is such a common pattern in our application that I ask myself how we could DRY this up? Maybe with a `loading` mixin that gives us a method wrapper and handles the update flags. :thinking: